src/frontend/komponenter/Oppgaver/OppgaveList.tsx

```diff
@@ -28,223 +29,247 @@ interface IOppgaveListProps {
-                                    <div className={'oppgavetype'}>
-                                        <Lenke href="#" onClick={() => onColumnSort('oppgavetype')}>
-                                            Oppgavetype
-                                        </Lenke>
-                                    </div>
-                                </th>
-                                <th className={'gjelder'}>
-                                    <Lenke href="#" onClick={() => onColumnSort('behandlingstema')}>
-                                        Gjelder
-                                    </Lenke>
-                                </th>
-                                <th className={'frist'}>
-                                    <Lenke
-                                        href="#"
-                                        onClick={() => onColumnSort('fristFerdigstillelse')}
+        <div className={'oppgavelist'}>
+            <div className={'oppgavelist__header'}>
+                <Systemtittel className={'oppgavelist__header__tittel'}>
+                    Oppgaveliste - visning
+                </Systemtittel>
+                <OppgavelisteNavigator />
+            </div>
+            <div>
+                <table className="tabell">
+                    <thead className="tabell__head">
+                        <tr className="tabell__head__tr">
+                            {Object.values(ListFelt).map(felt => {
+                                return feltSortOrder.get(felt) ? (
+                                    <th
+                                        role="columnheader"
+                                        aria-sort={getAriaSort(felt)}
+                                        className={getSortLenkClassName(felt)}
                                     >
-                                        Frist
-                                    </Lenke>
-                                </th>
-                                <th className={'prioritet'}>
-                                    <Lenke href="#" onClick={() => onColumnSort('prioritet')}>
-                                        Prioritet
-                                    </Lenke>
-                                </th>
-                                <th>
-                                    <div className={'beskrivelse'}>Beskrivelse</div>
-                                </th>
-                                <th className={'bruker'}>Bruker</th>
-                                <th>
-                                    <div className={'enhet'}>
-                                        <Lenke
-                                            href="#"
-                                            onClick={() => onColumnSort('tildeltEnhetsnr')}
+                                        <div
+                                            className={
+                                                'oppgavelist__tabell__' + oppgaveFeltMap.get(felt)
```

Comment:
```suggestion
                                                'oppgavelist__tabell--' + oppgaveFeltMap.get(felt)
```

Here is a quick guide to bem: https://blog.decaf.de/2015/06/24/why-bem-in-a-nutshell/

We will use an util for this soon.

Reply:
I have different view here. Because in BEM, the last part is modifier, like 'hidden', 'hove', but here it is the element (E), e.g. oppgavelist__tabell__behandlingstema. What do you think about it?

Reply:
Ok, but then maybe a compromise is to join tabell-felt?